src/bikeracks/schema.sql

```diff
@@ -54,6 +54,29 @@ CREATE TABLE votes (
 /* want to be able to look up vote by a user for a particular rack*/
 CREATE UNIQUE INDEX rack_vote ON votes (user_id, rack_id);
 
+
+
+
+
+
```

Comment:
Why all this empty space?

Reply:
I just needed it to feel less cluttered. I can get rid of the space later.